pkg/operator/resource_reconciler.go

```diff
@@ -65,7 +64,7 @@ type ReconcilerMetrics interface {
 // ResourceReconciler will trigger object and status reconciliations based on
 // the events received from the informer.
 type ResourceReconciler struct {
-	logger log.Logger
+	logger *slog.Logger
```

Comment:
Any special reason to change this to a pointer?

Reply:
or maybe you confused *slog.Logger with *log.Logger?